k8s/charts/rtsp-camera-simulator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Create a separate PR for everything related to the RTSP camera simulator.

Reply:
deleted